### PR TITLE
Account info refactor

### PR DESCRIPTION
**Pull request by gyurka00**

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit 59740a26f9552d67781c5c3120bfd63680cbcd98

https://app.netlify.com/sites/augmint/deploys/5b27e9281f12b75027071d5d

File: src/components/AccountInfo.js [hunk 1]
```diff
@@ -18,17 +18,24 @@ export class AccountInfo extends React.Component {
                     userBalancesIsLoading
                 }
                 header={header}
+                style={{ padding: "0px 10px", border: "1px solid #ffad00", borderRadius: "0px 0px 5px 5px"}}
```
Comment (reyrodrigez) [inline src/components/AccountInfo.js]: ideally these all should live in the style.js

Comment (admrid) [reply src/components/AccountInfo.js]: in first round i fixed white on white balace nums & copy icon

Comment (admrid) [reply src/components/AccountInfo.js]: second round:
added the orange bg to the account info inside the topnav
coming up: third round w separators

**Pull request merged**